api/spec/v1.yaml

```diff
@@ -929,6 +929,20 @@ components:
       type: string
       enum: [emerald, sapphire, cipher]
 
+    List:
+      type: object
+      required: [total_count, is_total_count_clipped]
+      properties:
+        total_count:
```

Comment:
I also wonder if the block height at query time is important enough to return.

Reply:
Interesting question, it applies to almost every single response we generate. I guess "not for now" -- it would be clumsy to surface this in the UI, or to force the UI to refetch all data if it performs multiple fetches from the indexer and they end up being generated at different heights.